### DISCUSSION TITLE
Questions about eligibility

### DISCUSSION THREAD

**Discussion by johnsonjh** [Homebrew/discussions#3448] [Tap maintenance and Homebrew development] [answered]
Greetings ...

In my efforts to contribute formulas to get the few remaining things I absolutely depend on but aren't packaged in Brew, I've come across a case and I'm not sure how it would be handled.

I am the current maintainer of [DUMA](https://duma.sourceforge.io/). I am planning to get a new release out soon, a release I'd like to get into Brew.

This is a 'mature' project, but one with a long and storied history beginning in the 1980's, and has been the subject of [magazine articles](https://duma.sourceforge.io/74_75_76_77_78_Debugging_DE.pdf), and other press.  It is also currently packaged by Debian, Ubuntu and other [major distributions](https://repology.org/project/duma/versions).  It's a component of some other major projects, though without much fanfare.

The [SourceForge home page](https://duma.sourceforge.io/) clearly redirects to the [relatively new GitHub home](https://github.com/johnsonjh/duma), allowing easy verification that the GitHub project is the new canonical upstream.. however, the new GitHub repo has "only" 50-60 stars. If I worked to package this for Homebrew, would it be eligible for an exception to the notability guidelines?

While I'm sure I could simply ask some users and get enough stars to pass the test, that seems really "dirty" somehow to me!

Thanks,

**Comment by alebcay** [comment, marked as answer]
Generally speaking, the notability guidelines are flexible - their reason for being is more to keep out freshly minted (and likely soon abandoned) pet projects. I think it would be helpful to have some more maintainers chime in here but generally that is how I approach the topic of notability here.

In any case, it wouldn't hurt to go ahead and write a formula for DUMA. [Homebrew allows for the creation of "taps"](https://docs.brew.sh/Taps), which are third-party repositories (a la PPAs in Debian/Ubuntu-land), which you could also use to host a DUMA formula in the case that it's not accepted for a while on this issue (and is generally a great solution for hosting stuff that you want people to be able to have through Homebrew that may not be a good fit for inclusion in the main repository).

**Reply by johnsonjh** [reply]
Thanks - I think I will.  DUMA has the opposite "problem" than what you seem worried about - it's a program that's more than 35 years old and perhaps it's popularity has faded, but it still is a capable - if not the best - tool for certain jobs, and popular with embedded systems, being available as an option for toolchains built by crosstool-ng and as a component of T2SDE, for example.

I'll see if the new repo gains popularity by the time of the next release, and it will hopefully be a non-issue by then. 

Thanks

**Reply by carlocab** [reply]
For me, our notability criterion is meant to screen out:

1. Weekend projects that are likely to fall into disrepair fairly soon
2. Software that with too few users to justify the maintenance burden

DUMA is certainly not the first kind, and appears unlikely to be the second, so I personally wouldn't be opposed to adding it to homebrew-core.

We're also aware of some older projects that only recently moved to GitHub and hence typically don't have enough stars/forks/etc. We're usually willing to look past our notability requirement in those cases.
